test/examples/ucp_client_server.c

```diff
@@ -189,6 +189,11 @@ static void print_result(int is_server, char *recv_message)
  */
 static void request_wait(ucp_worker_h ucp_worker, test_req_t *request)
 {
+    /*  if operation was completed immediately */
+    if (request == NULL){
```

Comment:
looks like there is missed space between `)` and `{`, should be
```
if (request == NULL) {
```

Reply:
fixed